srcalc/src/main/java/gov/va/med/srcalc/web/ExceptionlessValueVisitor.java

```diff
@@ -0,0 +1,31 @@
+package gov.va.med.srcalc.web;
+
+import gov.va.med.srcalc.domain.variable.BooleanValue;
+import gov.va.med.srcalc.domain.variable.DiscreteNumericalValue;
+import gov.va.med.srcalc.domain.variable.MultiSelectValue;
+import gov.va.med.srcalc.domain.variable.NumericalValue;
+import gov.va.med.srcalc.domain.variable.ProcedureValue;
+import gov.va.med.srcalc.domain.variable.Value;
+
+public abstract class ExceptionlessValueVisitor implements ValueVisitor
```

Comment:
I appreciate the Exceptionless version, but it definitely needs some Javadoc explanation.